scapy/layers/tls/automaton_cli.py

```diff
@@ -96,13 +97,12 @@ def parse_args(self, server="127.0.0.1", dport=4433, server_name=None,
                 self.advertised_tls_version = v
 
         self.linebreak = False
-        if isinstance(data, str):
+        if isinstance(data, bytes):
             self.data_to_send = [data]
+        if isinstance(data, six.string_types):
+            self.data_to_send = [raw(data)]
```

Comment:
This is duplicated as the above one

Reply:
Nope, six.string_types is `(basestring,)` in Python 2 and `(str,)` in Python 3. Anyway I've used an `elif` statement as it makes more sense here.